### DISCUSSION TITLE
Release 1.2.0 Beta 15

### DISCUSSION THREAD

**Discussion by monkeyman4000** [apohl79/audiogridder#1236] [General]
Hi

Is it still possible to download Release 1.2.0 Beta 15. 

The problem I am having is that with the latest release when it scans the plugins it finds them and is in the list. But when trying to load up in the plugin, they are not available. beta 15 was working fine and no problems at all.

**Comment by apohl79** [comment]
you can access old versions here https://audiogridder.com/releases/
